src/client/src/rt-components/popup/Popup.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { MouseEvent } from 'react'
+import { PopupContainer, PopupPanel, Body } from './styled'
+
+interface Props {
+  className?: string
+  open?: boolean
+  onClick?: (e: MouseEvent<HTMLDivElement>) => void
+}
+
+export default class Popup extends React.Component<Props> {
+  constructor(props: Props) {
+    super(props)
+    this.onClick = this.onClick.bind(this)
+  }
+
+  onClick(e: MouseEvent<HTMLDivElement>) {
```

Comment:
If you want, you can save yourself some syntax by:
`onClick = (e) =>{}`
(I know you know this already)

Reply:
Yep, I copied this code from modal and made minimal changes. In modal, it's called "onDismiss". But I can change it in Popup (and Modal) if you'd like!

Reply:
I changed it in Popup only.